recipes/raven-assembler/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = 'raven-assembler' %}
+{% set version = '0.0.0' %}
+{% set sha256 = '6ca62a0152e130216da2959099ca152aa21d6758770b74c430f515ff755c1b2d' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  url: https://github.com/lbcb-sci/raven/releases/download/0.0.0/raven-v{{ version }}.tar.gz
```

Comment:
HI,

You could replace 0.0.0 by {{ version }}

Reply:
Thanks, I replaced as suggested